app/models/page.rb

```diff
@@ -161,22 +157,39 @@ def basename
     @name.to_s.gsub(/[^0-9a-zA-Z\-\_\/]/, '').underscore
   end
 
-  def metadata
-    defaults = {
-      # Default to rendering table of contents
-      "toc": true,
-      # Default to H3s being included in the table of contents
-      "toc_include_h3": true
-    }
-    if file.front_matter
-      defaults.merge(file.front_matter.symbolize_keys)
-    else
-      defaults
-    end
+  # Page description, either from front matter or extracted from the markdown
+  def description
+    front_matter.fetch(:description, extracted_data.fetch("shortDescription"))
```

Comment:
Does this mean the frontmatter field is `description`? I wonder if it should be `meta_description` just so its purpose is clearer 🤔

Reply:
I think `description` is pretty clear and allows for other uses – for example, it's also used as the value of the `og:description` tag. Yeah ok, that's also a meta tag but you get the drift!